circle.yml

```diff
@@ -44,11 +45,8 @@ machine:
 
 checkout:
   post:
-    - |
-      git clone --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
-      cd /home/ubuntu/st2/st2-packages
-      git checkout ${CIRCLE_BRANCH} || true
```

Comment:
See the discussion why this order `|| true` hack is there:
https://github.com/StackStorm/st2/pull/2569#discussion_r55524503